cmd/utils/flags.go

```diff
@@ -554,6 +558,11 @@ var (
 		Usage: "HTTP-RPC server listening port",
 		Value: node.DefaultHTTPPort,
 	}
+	HTTPAuthPortFlag = cli.IntFlag{
+		Name:  "http.authport",
```

Comment:
Since this is apparently used by both HTTP as well as websocket, please use `rpc.authport`.